src/client/Item/ItemDefinition.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Items in Cosmos DB are simply JSON objects.
  * Most of the Item operations allow for your to provide your own type
- * that extends the very simple ItemDefintion.
+ * that extends the very simple Itemdefinition.
```

Comment:
```suggestion
 * that extends the very simple ItemDefinition.
```
I think we want to keep this one the same to match the type name

Reply:
Good thinking!